### PR TITLE
Updates README with BEHAVIOR instructions

### LINKED ISSUE #274: BEHAVIOR: Add instructions for installing and running BEHAVIOR code to README
Something similar to [this](https://github.com/Learning-and-Intelligent-Systems/iGibson/blob/master/mac_behavior_installation.md)

### PR DESCRIPTION
**Pull request by NishanthJKumar**
* also minor change in actual behavior.py
Resolves #274

### CODE REVIEW

**Review by tomsilver — APPROVED** [review]
thank you!

File: README.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ A simple implementation of search-then-sample task and motion planning is provid
 * This repository uses Python versions 3.8+.
 * Run `pip install -r requirements.txt` to install dependencies.
 
+### (Optional) BEHAVIOR Installation
+This repository and some of the functionality it enables are integrated with the [BEHAVIOR benchmark of tasks](https://behavior.stanford.edu/benchmark-guide) simulated with the [iGibson simulator](https://github.com/StanfordVL/iGibson). To install iGibson with BEHAVIOR, follow [these instructions](https://stanfordvl.github.io/behavior/installation.html). Importantly, note that you need to use our forks of the [iGibson](https://github.com/Learning-and-Intelligent-Systems/iGibson) and [bddl](https://github.com/Learning-and-Intelligent-Systems/bddl) repositories. Thus, instead of steps 1a. and 1b. in the linked instructions, run:
+```
+https://github.com/Learning-and-Intelligent-Systems/iGibson.git
```
Comment (ronuchit) [inline README.md:30]: give the full command that people need to run (for instance, i notice there's a `--recursive` flag in the actual installation, do we need that?

File: README.md [hunk 2]
```diff
@@ -24,6 +24,13 @@ A simple implementation of search-then-sample task and motion planning is provid
 * This repository uses Python versions 3.8+.
 * Run `pip install -r requirements.txt` to install dependencies.
 
+### (Optional) BEHAVIOR Installation
+This repository and some of the functionality it enables are integrated with the [BEHAVIOR benchmark of tasks](https://behavior.stanford.edu/benchmark-guide) simulated with the [iGibson simulator](https://github.com/StanfordVL/iGibson). To install iGibson with BEHAVIOR, follow [these instructions](https://stanfordvl.github.io/behavior/installation.html). Importantly, note that you need to use our forks of the [iGibson](https://github.com/Learning-and-Intelligent-Systems/iGibson) and [bddl](https://github.com/Learning-and-Intelligent-Systems/bddl) repositories. Thus, instead of steps 1a. and 1b. in the linked instructions, run:
```
Comment (ronuchit) [inline README.md:28]: those aren't steps 1a and 1b, they're steps 2a and 2b of Option 1

File: README.md [hunk 3]
```diff
@@ -24,6 +24,13 @@ A simple implementation of search-then-sample task and motion planning is provid
 * This repository uses Python versions 3.8+.
 * Run `pip install -r requirements.txt` to install dependencies.
 
+### (Optional) BEHAVIOR Installation
```
Comment (ronuchit) [inline README.md:27]: clarify that you should follow Option 1 on the linked instructions

File: README.md [hunk 4]
```diff
@@ -42,6 +49,15 @@ Please make sure to `export PYTHONHASHSEED=0` when running the code. You can add
 * Monitor with `squeue -u ronuchit`, or cancel jobs with `scancel -u ronuchit` (standard Slurm commands).
 * When all jobs are done, run `python analysis/analyze_supercloud_experiments.py` (still on supercloud) to print out the results table.
 
+### Running Experiments on BEHAVIOR
+* Currently, only the `oracle` approach is implemented to integrate with BEHAVIOR
+* Note that you'll probably want to provide the `--timeout 1000`, `--max_num_steps_check_policy 1000` to prevent early stopping.
```
Comment (ronuchit) [inline README.md:54]: "provide the" -> "provide the command line arguments"

File: README.md [hunk 5]
```diff
@@ -42,6 +49,15 @@ Please make sure to `export PYTHONHASHSEED=0` when running the code. You can add
 * Monitor with `squeue -u ronuchit`, or cancel jobs with `scancel -u ronuchit` (standard Slurm commands).
 * When all jobs are done, run `python analysis/analyze_supercloud_experiments.py` (still on supercloud) to print out the results table.
 
+### Running Experiments on BEHAVIOR
+* Currently, only the `oracle` approach is implemented to integrate with BEHAVIOR
```
Comment (ronuchit) [inline README.md:53]: end all lines with periods (for consistency)

File: README.md [hunk 6]
```diff
@@ -42,6 +49,15 @@ Please make sure to `export PYTHONHASHSEED=0` when running the code. You can add
...
+* Note that you'll probably want to provide the `--timeout 1000`, `--max_num_steps_check_policy 1000` to prevent early stopping.
+* Set `--option_model_name behavior_oracle` to use the behavior option model and speed up planning by a significant factor
+* Set `--behavior_task_name` to the name of the particular bddl task you'd like to run (e.g. `re-shelving_library_books`)
+* Set `--behavior_scene_name` to the name of the house setting (e.g. `Pomaria_1_in`, or `Rs_int`) you want to try running the particular task in.
```
Comment (ronuchit) [inline README.md:57]: have we tested Rs_int btw?

File: README.md [hunk 7]
```diff
@@ -42,6 +49,15 @@ Please make sure to `export PYTHONHASHSEED=0` when running the code. You can add
...
+* `--behavior_randomize_init_state` can be set to True if you want to generate multiple different initial states that correspond to the BDDL init conditions of a particular task.
+* Example command: `python src/main.py --env behavior --approach oracle --seed 0 --timeout 1000 --max_samples_per_step 20 --behavior_mode simple --max_num_steps_check_policy 1000 --option_model_name behavior_oracle --num_train_tasks 2 --num_test_tasks 2 --behavior_randomize_init_state True --behavior_scene_name Pomaria_1_int --behavior_task_name re-shelving_library_books`
```
Comment (ronuchit) [inline README.md:59]: what does the behavior_mode simple do? Do we need to explain that one?

**Review by ronuchit — CHANGES_REQUESTED** [review]
looks good, couple points of clarification

File: README.md (hunk 1 above)
Comment (NishanthJKumar) [reply README.md:30]: woops; forgot about this, good catch!

File: README.md (hunk 6 above)
Comment (NishanthJKumar) [reply README.md:57]: yes, but not with `re-shelving_library_books` (some tasks exist only in some houses). I'll mention something to this effect

**Review by ronuchit — APPROVED** [review]
lgtm thanks!!

**Pull request merged**